ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -40,6 +40,7 @@ class DashboardPage extends Component {
       selectedCell: null,
       isTemplating: false,
       zoomedTimeRange: {zoomedLower: null, zoomedUpper: null},
+      manualRefresh: Date.now(),
```

Comment:
This can start as `null`

Reply:
hmm... I disagree. Although we do have a default prop-- good idea, btw-- this will be the first value of the key. I don't think the initial key value should be 0.